test/esinstall/config-custom-path/config-custom-path.test.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const {
+  existsPackageJson,
+  runTest,
+  testLockFile,
+  testWebModules,
+} = require('../esinstall-test-utils.js');
+
+require('jest-specific-snapshot'); // allows to call expect().toMatchSpecificSnapshot(filename, snapshotName)
+
+describe('config-custom-path', () => {
+  beforeAll(() => {
+    // Needed so that ora (spinner) doesn't use platform-specific characters
+    process.env.CI = '1';
```

Comment:
Ah I forgot about this! We’re actually not using ora anymore so it looks like we could have dropped this before.

But in general, I like the approach you took here of having this live in the tests specifically rather than globally, because this is specific to the type of test, and may be modified/removed.

Reply:
These have all been removed.